Lib/sre_constants.py

```diff
@@ -21,6 +21,17 @@
 # should this really be here?
 
 class error(Exception):
+    '''Exception raised for invalid regular expressions.
+
+    The error instance has the following additional attributes:
+
+    msg: The unformatted error message
+    pattern: The regular expression pattern
+    pos: The index of pattern where compilation failed
```

Comment:
The `re` module documentation is copied from the `json` module documentation, which was copied from the `simplejson` documentation. If change the wording it makes sense to change it consistently in all these places.